commons/src/main/java/fr/free/nrw/commons/category/CategorizationFragment.java

```diff
@@ -22,35 +22,43 @@
 import fr.free.nrw.commons.upload.MwVolleyApi;
 
 import java.io.IOException;
+import java.sql.ResultSet;
 import java.util.ArrayList;
 import java.util.Date;
 import java.util.HashMap;
 import java.util.HashSet;
+import java.util.LinkedHashSet;
 import java.util.List;
+import java.util.Set;
+import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.ScheduledThreadPoolExecutor;
 
+
 public class CategorizationFragment extends SherlockFragment{
     public static interface OnCategoriesSaveHandler {
         public void onCategoriesSave(ArrayList<String> categories);
     }
 
     ListView categoriesList;
-    EditText categoriesFilter;
+    protected EditText categoriesFilter;
```

Comment:
I don't know why all or most of these members aren't private unless they're used in tests. It might be worth adding a hygiene patch later.